cmd/root.go

```diff
@@ -14,13 +15,27 @@ var rootCmd = &cobra.Command{
 
 It is a versatile bot that can seamlessly integrate with multiple data sources, such as GitHub, and soon Phylum,
 Vulnbot empowers developers and security teams to efficiently manage and respond to security threats.`,
+	PersistentPreRun: func(cmd *cobra.Command, args []string) {
+		quiet, _ := cmd.Flags().GetBool("quiet")
+		verbosity, _ := cmd.Flags().GetCount("verbose")
+
+		if quiet == true {
+			logger.SetLogLevel(zerolog.Disabled)
+		}
+		if verbosity > 0 {
```

Comment:
[q] I understand the `MarkFlagsMutuallyExclusive` is applied at the CLI level, should we also make this an `else if` to signal that `quiet` takes precedence if somehow both are passed in programmatically?

Reply:
Oh yeah good call. I had meant to do that and just never got back to it!